src/tab_container.js

```diff
@@ -0,0 +1,241 @@
+    container.appendChild(form);
+    const tasks = project.projectTask;
+    let editedTask = '';
+    let task = '';
+    let taskToBeEdited = '';
+    for (let i = 0; i < tasks.length; i += 1) {
+      task = tasks[i];
+      if (taskID === task.taskId) {
+        taskToBeEdited = tasks[i];
+        title.value = task.taskTitle;
+        description.value = task.taskDescription;
+        dueDate.value = task.TaskDueDate;
+        priority.value = task.TaskPriority;
+        SaveTaskButton.addEventListener('click', () => {
+          editedTask = {
+            taskId: taskToBeEdited.taskId,
+            taskTitle: title.value,
+            taskDescription: description.value,
+            TaskDueDate: dueDate.value,
+            TaskPriority: priority.value,
+          };
+          taskToBeEdited = editedTask;
+          project.projectTask[i] = editedTask;
+          showTaskInProject(project);
+        });
+      }
+    }
+  };
+
+  const DeleteTask = (project, taskID) => {
+    const tasks = project.projectTask;
+    for (let i = 0; i < tasks.length; i += 1) {
+      const task = tasks[i];
+      if (taskID === task.taskId) {
+        tasks.splice(i, 1);
+        break;
+      }
+    }
+    return project;
+  };
+
+  const showCreateTaskButton = (obj) => {
+    AddTaskButton.addEventListener('click', () => {
+      createTaskForm(obj);
+      AddTaskButton.style.display = 'none';
+    });
+
+    container.appendChild(document.createElement('br'));
+    container.appendChild(document.createElement('br'));
+    container.appendChild(AddTaskButton);
+  };
+
+  const createTaskForm = (obj) => {
+    const previousForm = container.querySelector('form');
+    if (previousForm) previousForm.remove();
+    const form = document.createElement('form');
+    form.appendChild(document.createElement('br'));
+    form.appendChild(document.createElement('br'));
+    const x = document.createElement('LABEL');
+    const t = document.createTextNode('Task Title: ');
+    x.setAttribute('for', 'male');
+    x.appendChild(t);
+    form.append(x);
+    const title = document.createElement('INPUT');
+    title.setAttribute('type', 'text');
+    form.appendChild(title);
+
+    form.appendChild(document.createElement('br'));
+    form.appendChild(document.createElement('br'));
+    const x1 = document.createElement('LABEL');
+    const t1 = document.createTextNode('Description: ');
+    x1.appendChild(t1);
+    form.append(x1);
+    const description = document.createElement('INPUT');
+    description.setAttribute('type', 'text');
+    form.appendChild(description);
+
+    form.appendChild(document.createElement('br'));
+    form.appendChild(document.createElement('br'));
+    const x2 = document.createElement('LABEL');
+    const t2 = document.createTextNode('Due Date: ');
+    x2.appendChild(t2);
+    form.append(x2);
+    const dueDate = document.createElement('INPUT');
+    dueDate.setAttribute('type', 'text');
+    form.appendChild(dueDate);
```

Comment:
## Optional 
- [ ] It would be great if this field would be implemented with the `Date()` method from [JavaScript ](https://www.w3schools.com/js/js_dates.asp)